lib/migrate/Migrator.js

```diff
@@ -278,10 +278,18 @@ class Migrator {
   forceFreeMigrationsLock(config) {
     this.config = getMergedConfig(config, this.config);
 
-    const lockTable = getLockTableName(this.config.tableName);
+    const lockTableName = getLockTableName(this.config.tableName);
+
     return getSchemaBuilder(this.knex, this.config.schemaName)
       .hasTable(lockTable)
-      .then((exist) => exist && this._freeLock());
+      .then((exist) => {
+        if (exist) {
+          await getTable(this.knex, lockTableName, this.config.schemaName).del();
```

Comment:
can you rewrite whole chain to be async/await?

Reply:
Sure, I could do that.